src/index.d.ts

```diff
@@ -453,220 +453,220 @@ declare namespace admin.database.ServerValue {
 }
 
 type BaseMessage = {
```

Comment:
Shouldn't be readonly. Same goes for all the types referenced here, and their child types (e.g. AndroidConfig -> AndroidNotification)